model/database/spans.yaml

```diff
@@ -838,3 +838,53 @@ groups:
       - ref: azure.cosmosdb.operation.contacted_regions
         requirement_level:
           conditionally_required: If available.
+  - id: span.db.oracledb.client
+    type: span
+    stability: development
+    extends: span.db.sql.client
+    span_kind: client
+    brief: >
+      Attributes for Oracle Database
+    attributes:
+      - ref: db.namespace
+        sampling_relevant: true
+        brief: >
+          The database associated with the connection, qualified by the instance name,
+          database name and service name.
+        note: |
+          `db.namespace` SHOULD be set to the combination of instance name, database name and
+          service name following the `{instance_name}|{database_name}|{service_name}` pattern.
```

Comment:
any reason not to follow the same pattern as MS SQL? (use `.` as a separator)?

https://github.com/open-telemetry/semantic-conventions/blob/e29f8dd779832b6011437e5e49288505ec8fe19c/model/database/spans.yaml#L128

I.e. `{instance_name},{database_name}`,

also check out my other comment on `service_name` - it should probably be in the `server.address` and don't need to be in the namespace

Reply:
it sounds good, please keep the `|`.

I'm leaving this discussion unresolved for other reviewers to see it

Reply:
@alanwest @lmolkova what do you think of changing the sqlserver separator from `.` to `|`?

Reply:
resolving discussion, I opened an issue to discuss / decide this for sqlserver